org.flameshot.Flameshot.yml

```diff
@@ -1,26 +1,23 @@
 app-id: org.flameshot.Flameshot
 runtime: org.kde.Platform
-runtime-version: 5.15-21.08
+runtime-version: 5.15-23.08
 sdk: org.kde.Sdk
 command: flameshot
 finish-args:
-  # X11 + XShm access
+  # Fallback X11 + XShm access
   - --share=ipc
-  - --socket=x11
```

Comment:
Crashes immediately on GNOME wayland

```
qt.qpa.xcb: could not connect to display 
qt.qpa.plugin: Could not load the Qt platform plugin "xcb" in "" even though it was found.
This application failed to start because no Qt platform plugin could be initialized. Reinstalling the application may fix this problem.

Available platform plugins are: eglfs, minimal, minimalegl, offscreen, vnc, wayland-egl, wayland, wayland-xcomposite-egl, wayland-xcomposite-glx, xcb.
```

Does Flameshot still default to X11? `flatpak run --env=QT_QPA_PLATFORM=wayland org.flameshot.Flameshot` makes no difference.

Seems the only thing works is to block fallback-x11 and allow both x11 and wayland

Reply:
or remove the wayland socket. It seems that x11 is forced, and given how Qt5 is bugpalooza on Wayland, it's not surprising.

See upstream:
https://github.com/flameshot-org/flameshot/blob/0bbb9528615c1d3697e0538c4a53d8d0e00ade0a/src/main.cpp#L42

Reply:
Qt has a patch to do that same kind of things based on the session but the KDE runtime patch it out. That's Qt way of fixing wayland issue with some compositors.

Reply:
Yea, @pm4rcin can you revert it to as it was ie. `x11` only?

Reply:
Done.